### PR TITLE
Add component files and test files

### PR DESCRIPTION
**Pull request by kaohman**
Set up files for each component in the app.

### CODE REVIEW

File: src/Button.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+
+  render() {
+    return (
+      
+    );
+  }
+}
+
+export default App;
```
Comment (trbachmann) [inline src/Button.js:19]: Should this be export default Button?

File: src/Map.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+
+  render() {
+    return (
+      
+    );
+  }
+}
+
+export default App;
```
Comment (trbachmann) [inline src/Map.js:22]: I think this line should be export default Map

**Review by trbachmann — CHANGES_REQUESTED** [review]
Thank you for putting the file structure together and adding in Leaflet! Will you please edit the export default line at the bottom of the files?

**Comment by kaohman** [comment]
Yes, thanks for catching that! I will make those changes now and make a new PR.

**Pull request closed without merging**